build.sh

```diff
@@ -88,6 +99,9 @@ build() {
   if [[ $REPLACE_KEYS = 1 ]]; then
     replace_keys
   fi
+  if [[ $NO_SECURE_BOOT = 1 ]]; then
```

Comment:
NO_SECURE_BOOT is confusing as it does not do what it means

Reply:
Changed to SECURE_BOOT_DEFAULT_DISABLED